content/03-reference/01-tools-and-interfaces/01-prisma-schema/index.mdx

```diff
@@ -150,40 +156,53 @@ DATABASE_URL=postgresql://test:test@localhost:5432/test?schema=public
 
 When running any command that needs to access the database defined via the `datasource` block (e.g. `prisma introspect`), the Prisma CLI automatically loads the `DATABASE_URL` environment variables from the `.env` file and makes it available to the CLI.
 
-If you want environment variables to be evaluated at runtime, you need to load them manually in your application code, e.g. using [`dotenv`](https://github.com/motdotla/dotenv):
+### Using environment variables in your code
+
+If you want environment variables to be evaluated at runtime, you need to load them manually in your application code (for example, by using [`dotenv`](https://github.com/motdotla/dotenv)):
 
 ```ts
 import * as dotenv from 'dotenv'
 
-dotenv.config() // load the environment variables
+dotenv.config() // Load the environment variables
 console.log(`The connection URL is ${process.env.DATABASE_URL}`)
 ```
 
-### Use multiple .env files or .env files outside the prisma folder
+### Manage `.env` files with `dotenv-cli`
 
-To use multiple `.env` files:
+- Use multiple `.env` files
+- Use `.env` files outside the `prisma` folder
```

Comment:
The correct wording would probably be "not besides your Prisma schema" or "not in the location where the CLI will look by itself" - see the table above